streams/shared/src/main/scala/scalaz/stream/ZStream.scala

```diff
@@ -660,6 +660,16 @@ trait ZStream[-R, +E, +A] extends Serializable { self =>
         }
       }
   }
+
+  /**
+   * @note in case you need extreme performance, make sure to use capacity
+   *   which is a power of 2.
+   * Stores this stream to a fixed managed queue and streams data from the managed queue.
```

Comment:
Let's emphasize the purpose of the combinator rather than its implementation: "Allow a faster producer to progress independently of a slower consumer by buffering up to `capacity` elements in a queue."

Reply:
Done